sunpy/util/decorators.py

```diff
@@ -182,3 +183,37 @@ def deprecate(obj, message=message, name=name, alternative=alternative):
         return deprecate(message)
 
     return deprecate
+
+
+class add_common_docstring(object):
+    """
+    A function decorator that will append and/or prepend an addendum
+    to the docstring of the target function.
+
+
+    Parameters
+    ----------
+
+    append : `str`, optional
+        A string to append to the end of the functions docstring.
+
+    prepend : `str`, optional
+        A string to prepend to the start of the functions docstring.
```

Comment:
sorry, I forgot to mention, there should be an entry here for `kwargs`, to say that they get passed to format for append and prepend.

Reply:
@Cadair you ok with the addition for ```kwargs```?

Reply:
yeah

Reply:
Well since the CI wont be fixed soon(<1 minute) Ill merge this now.